### ISSUE TITLE
Optional parameter output

### ISSUE THREAD

**Issue by jklenzing** [jklenzing/sami2py#29]
**Is your feature request related to a problem? Please describe.**
Add options for various sami2 outputs to suppress or add potential parameters (Ni, Te, Nn, etc)

**Describe the solution you'd like**
Need to pass flags through to fortran to choose which outputs are required.

**Comment by jklenzing** [comment]
Progress made with merge of #41.  Neutral density is now an optional output.

**Comment by JonathonMSmith** [comment]
I think options for producing magnetic coordinates would be useful

**Comment by aburrell** [comment]
@JonathonMSmith I agree.  However, if I recall, this is not trivial.  Perhaps (brainstorming) it would be best to have this as an outside utility, allowing users to convert output in the SAMI grid to different magnetic coordinate systems after the fact.  It would also be useful to have geographic coordinates.
